app/js/containers/MapContainer.jsx

```diff
@@ -16,15 +20,38 @@ class MapContainer extends Component {
     });
   }
 
+  componentDidMount() {
+    if (!this.props.licenseOwners || !this.props.licenseOwners.length) {
```

Comment:
Shall we not always do it in case the list change while you navigate?

Reply:
Yeah your right